source/dsl/DangerDSL.ts

```diff
@@ -16,14 +17,19 @@ export interface DangerDSLType {
    *  The GitHub metadata.
    */
   readonly github: Readonly<GitHubDSL>
+
+  /**
+   * Danger utils
+   */
+  readonly utils: Readonly<DangerUtilsDSL>
 }
 
 /* END FLOWTYPE EXPORT */
 
 export class DangerDSL {
   public readonly github: Readonly<GitHubDSL>
 
-  constructor(platformDSL: any, public readonly git: GitDSL) {
+  constructor(platformDSL: any, public readonly git: GitDSL, public readonly utils: DangerUtilsDSL) {
```

Comment:
I think as-is feels more consistent (which is probably why you went this way in the first place ) - otherwise there's a lot of DI